spec/features/visitor/login_spec.rb

```diff
@@ -19,14 +19,15 @@
   end
 
   it "if I am a merchant user I'm redirected to my merchant dashboard page when I submit valid information" do
-    merchant = User.create!(name: "Steve",
-                        address:"123 Main St.",
-                        city: "Fort Collins",
-                        state: "GA",
-                        zip: "66666",
-                        email: "chunky_merchant@example.com",
-                        password: "123password",
-                        role: 1)
+    meg = Merchant.create!(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+    merchant = meg.users.create!(name: "Steve",
```

Comment:
old test failing because we just had a merchant user that is now being redirected to a merchant dashboard but was created with no merchant. added a merchant and we're good to go.